src/components/ItemDetail/BorrowDialog.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <dialog-template @close="close">
+    <div :class="$style.container">
+      <h2 :class="$style.title">物品を借りる</h2>
+      <borrow-dialog-select
+        :details="details"
+        :selected-owner-name="selectedOwnerName"
+        @select-owner="selectOwner"
+      />
+      <div :class="$style.label">目的:</div>
+      <textarea v-model="purpose" rows="10" :class="$style.input" />
+      <div :class="$style.label">返却日:</div>
+      <input v-model="dueDate" type="date" :class="$style.input" />
+      <div v-if="owner && owner.count !== 1">
+        <div :class="$style.label">個数:</div>
+        <input
+          v-model="count"
```

Comment:
```suggestion
          v-model.number="count"
```
https://v3.vuejs.org/guide/forms.html#number

これないとリクエストが文字列で飛んじゃって400する